float64 x_d
float64 x
float64 y_d
float64 y
float64 z_d
float64 z
float64 yaw_d
float64 yaw